build.psm1

```diff
@@ -920,7 +922,23 @@ function Start-PSPester {
         }
         else
         {
-            & $powershell -noprofile -c $Command
+            if($PassThru.IsPresent)
+            {
+                $passThruFile = [System.IO.Path]::GetTempFileName()
+                try {
+                    $Command += "|Export-Clixml -Path '$passThruFile' -Force"
+                    Start-NativeExecution -sb {& $powershell -noprofile -c $Command} | ForEach-Object { Write-Host $_}
+                    Import-Clixml -Path $passThruFile | Where-Object {$_.TotalCount}
```

Comment:
Looking at the code below, when TotalCount is 0 that means no test were run.
Will this where-object filter work as expected in this case (TotalCount is 0)?